pages/docs/contributing/hacktoberfest.mdx

```diff
@@ -2,7 +2,7 @@ import DocsLayout from "@components/layouts/DocsLayout.js";
 
 ## Hacktoberfest
 
-Participate this annual event which encourages people to contribute to Open Source throughout the month of October by contributing to this project.
+Participate in this annual event which encourages people to contribute to Open Source throughout the month of October by contributing to this project.
```

Comment:
```suggestion
Participate in this annual event by contributing to this project! 😄  
```

Reply:
Thanks Chrissy but I prefer that it has "open source" in the sentence and also helps for seo